components/nexusoperations/statemachine.go

```diff
@@ -78,9 +78,7 @@ func AddChild(node *hsm.Node, id string, event *historypb.HistoryEvent, eventTok
 			ScheduleToCloseTimeout: attrs.ScheduleToCloseTimeout,
 			RequestId:              attrs.RequestId,
 			State:                  enumsspb.NEXUS_OPERATION_STATE_UNSPECIFIED,
-			// TODO(bergundy): actually delete on completion if this is set.
-			DeleteOnCompletion:  deleteOnCompletion,
```

Comment:
Wanna also deprecate this in the protos?

Reply:
What does this entail? Just a "deprecaaated" comment in the proto file?

Reply:
You can mark it as `reserved`.